Add rebuildConfigs param and set configs dir via option in command line (#70)
yiisoft/yii-console: README.md

Since the package is based on [Symfony Console component](https://symfony.com/doc/current/components/console.html),
refer to its documentation for details on how to use the binary and create your own commands.

> ### Using alternative sets of configurations 

Reviewer: ```suggestion
### Using alternative set of configurations 
```

Use option `--config` (`-c`) for set name of alterntaive configuration:

```
vendor\bin\yii -ctest
vendor\bin\yii --config=test
``` 

> For more info about alternative sets of configurations and usage see its 

Reviewer: ```suggestion
For more info about alternative configuration sets their and usage see its 
```

[documentation](https://github.com/yiisoft/composer-config-plugin/blob/master/docs/en/alternatives.md).

> ### Parameters of console

Reviewer: ```suggestion
### Console parameters
```

##### rebuildConfig
 
Type: `bool|\Closure`

Default Value: 
```php
static fn() => getenv('APP_ENV') === 'dev'
```

> Force rebuild configuration before each run console.

Reviewer: ```suggestion
Force rebuild configuration before each run.
```
